platforms/documentation/docs/src/docs/userguide/dep-man/understanding-dependency-resolution/dependency_resolution_model.adoc

```diff
@@ -0,0 +1,266 @@
+At this point, <<#sec:resolution-strategy,version conflict resolution>> is resumed.
+
+=== B. Resolving module dependencies for a specific version
+
+When Gradle tries to resolve a required dependency with a specific version, it follows this process:
+
+1. **Repository inspection**: Gradle checks each repository in the order they are defined.
+- It looks for metadata files describing the module (`.module`, `.pom`, or `ivy.xml`), or directly for artifact files.
+- Modules with metadata files (`.module`, `.pom`, or `ivy.xml`) are prioritized over those with just an artifact file.
+- Once metadata is found in a repository, subsequent repositories are ignored.
+2. **Retrieving and parsing metadata**: If metadata is found, it is parsed.
+- If the POM file has a parent POM, Gradle recursively resolves each parent module.
+3. **Requesting artifacts**: All artifacts for the module are fetched from the same repository that provided the metadata.
+4. **Caching**: All data, including the repository source and any potential misses, are stored in the <<dependency_caching.adoc#sec:controlling-dynamic-version-caching,dependency cache>> for future use.
+
+[NOTE]
+====
+The point above highlights a potential issue with integrating <<supported_repository_types.adoc#sec:case-for-maven-local,Maven Local>>. Since Maven Local acts as a Maven cache, it may occasionally miss artifacts for a module. When Gradle sources a module from Maven Local and artifacts are missing, it assumes those artifacts are entirely unavailable.
+====
+
+[[sec:repository-disabling]]
+=== Repository disabling
+
+When Gradle fails to retrieve information from a repository, it disables the repository for the remainder of the build and fails all dependency resolution.
+
+This behavior ensures reproducibility.
+
+If the build were to continue while ignoring the faulty repository, subsequent builds could produce different results once the repository is back online.
+
+[[sub:http-retries]]
+==== HTTP Retries
+
+Gradle will attempt to connect to a repository multiple times before disabling it.
+If the connection fails, Gradle retries on specific errors that might be temporary, with increasing wait times between retries.
+
+A repository is blacklisted when it cannot be reached, either due to a permanent error or after the maximum number of retries has been exhausted.
+
+== Result. The Dependency tree
+
+Once the process is complete, a dependency tree is created.
+
+The dependency tree is a hierarchical representation of all the dependencies required by a project, including direct dependencies (declared explicitly) and transitive dependencies (pulled in automatically by those direct dependencies).
+The graph shows how dependencies relate to each other and how Gradle resolves them.
+
+For the dependency `org.jetbrains.kotlinx:kotlinx-serialization-json:1.5.1`, the graph includes the primary dependency and all its transitive dependencies.
+Here's what the graph looks like:
+
+[source,json]
```

Comment:
Should this example be collapsed by default?

Reply:
No, I want folks to see this by default.